### ISSUE TITLE
simplified domain of admins capability is the empty string

### ISSUE THREAD

**Issue by ael-code** [insomnia-lab/libreant#222]
regular domain is ```.+``` (match all).
simToReg(".+") is not capable to handle this string and return the empty string.

**Comment by boyska** [comment]
have you any clue about _why_ can't `simToReg` handle it?

**Comment by ael-code** [comment]
> have you any clue about why can't simToReg handle it?

Because it's not a *simplified regular expression*.

**Comment by boyska** [comment]
I can't understand the consequences of this bug

**Comment by ael-code** [comment]
We need to extend our *simplified regular expression* in order to support multilevel expansion. At the moment we only support special char `*` that is an intra-slashes expansion.

simToReg will convert `*` in `[^/]+`.
We need another special char to be converted in `.+`

**Comment by boyska** [comment]
no, really, I am missing something. Is there any bug? are there thing that do not work? are there things that a user cannot do?

**Comment by ael-code** [comment]
On 12/27/2015 04:01 PM, BoySka wrote:
> no, really, I am missing something. Is there any bug? are there thing
> that do not work? are there things that a user cannot do?
> 

It's not possible to create a capability with a domain that match all
strings after certain char.

For instance it's not possible to generate some domain like '.+' or
'^/?volumes/.+' passing through a *simplified expression*.

**Comment by boyska** [comment]
ael wrote:
>On 12/27/2015 04:01 PM, BoySka wrote:
>> no, really, I am missing something. Is there any bug? are there thing
>> that do not work? are there things that a user cannot do?
>
>It's not possible to create a capability with a domain that match all
>strings after certain char.
>
>For instance it's not possible to generate some domain like '.+' or
>'^/?volumes/.+' passing through a *simplified expression*.

oh, ok, now this is *much* clearer.

Why don't we fix at the UI layer? I mean, if a user wants to have
simplified expression (99% of the time that's what she wants) then
everything is fine. Clicking on a certain checkbox, normal regexp can be
inserted directly.
